code/modules/mob/living/basic/vermin/cockroach/cockroach.dm

```diff
@@ -48,8 +50,7 @@
 		minimum_survivable_temperature = 140 // 40kelvin below icebox temp
 		add_atom_colour("#66ccff", FIXED_COLOUR_PRIORITY)
 	. = ..()
-	var/static/list/roach_drops = list(/obj/effect/decal/cleanable/insectguts)
-	AddElement(/datum/element/death_drops, roach_drops)
+	AddElement(/datum/element/death_drops, death_drops)
```

Comment:
not using a static list for this kind of ruins the point of the element (as limited as it is), just ftr

Reply:
gave bloodroach a static list var instead